packages/arb-validator-core/ethbridge/arbRollupWatcher.go

```diff
@@ -76,10 +90,15 @@ func init() {
 	confirmedAssertionID = parsedRollup.Events["ConfirmedAssertion"].ID()
 
 	transactionMessageDeliveredID = parsedInbox.Events["TransactionMessageDelivered"].ID()
+	transactionMessageBatchDeliveredID = parsedInbox.Events["TransactionMessageBatchDelivered"].ID()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [line is 100 characters, out of limit 80](https://app.codacy.com/gh/OffchainLabs/arbitrum/pullRequest?prid=5428340)